src/openforms/registrations/contrib/objects_api/plugin.py

```diff
@@ -161,7 +160,59 @@ def register_submission(
                 get_drc=get_drc,
                 language=language_code_2b,
             )
-            record_data["csv_url"] = submission_csv_document["url"]
+            csv_url = submission_csv_document["url"]
+
+        base_render_context = {
+            "_submission": submission,
+            **get_variables_for_context(submission),
```

Comment:
Let's use the `variables` naming here, because that's what they are conceptually *and* technically. So a namespace `variables` (and not `submission.variables`).